gcovr.cfg

```diff
@@ -0,0 +1 @@
+exclude = _build-cov/python-cffi/netplan/_*
```

Comment:
I just realized that the test coverage-c is failing for me. The reason is because my build directory is called `build`, and not `_build-cov`... we will probably need to invert the logic here to not depend on the build directory name.

I tested this and it worked:

```
filter = src/*
filter = tests/ctests/*
```